source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml

```diff
@@ -58,13 +58,25 @@
         <div class="title">Plan de acción </div><textarea class="content-editable" data-bind="value: evaluation.actionPlanComment, enable: isEvaluationEditable()"></textarea>
       </div>
     </div>
-    <h4>Comentarios de la Evaluación</h4>
+    <h4>Comentarios de la Empresa</h4>
     <div class="comments-container comments-company-container">
       <textarea class="content-editable big-comment" data-bind="value: generalComment, enable: isEvaluationEditable()"></textarea>
     </div>
-    <h4>Comentarios del Evaluado</h4>
-    <div class="comments-container comments-company-container">
-      <textarea class="content-editable big-comment" data-bind="value: evaluatedComment, enable: evaluation.readyForDevolution && !evaluation.finished && userView == 3"></textarea>
+    <div class="evaluation-comments">
+      <h4>Comentarios de la Evaluación</h4>
+      <div>
+        <!-- ko foreach: califications-->
+        <div data-bind="if: show() && (owner < 2 ||(owner == 2 && comments))">
```

Comment:
Almost the same. The difference is that this will show comments if they are visible (`show()`) and if they belong to the evaluated employee, evaluators or responsible, because there are 2 extra califications in the list: Average calification and Company calification. Those are left apart